drivers/hmis/app/jobs/hmis/migrate_assessments_job.rb

```diff
@@ -7,8 +7,12 @@
 module Hmis
   class MigrateAssessmentsJob < BaseJob
     include Hmis::Concerns::HmisArelHelper
+    include NotifierConfig
+
     attr_accessor :data_source_id
 
+    queue_as ENV.fetch('DJ_LONG_QUEUE_NAME', :long_running)
```

Comment:
Is this a good idea? The job takes quite a lot to run.

In order to run it asap on the long-running queue, then, I would do `Hmis::MigrateAssessmentsJob.set(priority: 0).perform_later(...)`

Reply:
Definitely this should be on the long-running queue, so yeah, that's correct.  You probably don't need to set priority unless there's a bunch of stuff in the queue you want to skip.

Reply:
Ok. Thanks